.gitignore

```diff
@@ -0,0 +1,28 @@
+##########################################
+# READ BEFORE ALTERING THIS FILE
+#
+# Only files specific to this repo or that will be generated as part of using this repo should
+# be ignored here. Files that are specific to particular development environments or users
+# should be ignored in the global gitignore to ignore for all repos, or in .git/info/exclude
+# to ignore for just this repo.
+# 
+# Examples of appropriate files for each location:
+# This file:
+#    python pyc files
+#    python cache files
+#    test configuration and output, including coverage data
+#
+# Global gitignore
+#    Eclipse .settings, .project, and .pyproject files
+#    Mac .DS_store files
+#    VSCode .vscode directory
+#
+# .git/info/exclude
+#    Temporary code / notes while exploring new repo features
+#    Personal data used for manual testing
+#
+##########################################
+
+/.pytest_cache/
+__pycache__
+/.venv/
```

Comment:
do you need to add in the directories that get created when you run `uv build` so you don't get the build gubbins committed to the repo?

Reply:
I haven't built a package with uv yet so I didn't know about the gubbins, but when I do I'll update the ignore